tests/e2e/tidbcluster/across-kubernetes.go

```diff
@@ -159,6 +163,53 @@ var _ = ginkgo.Describe("[Across Kubernetes]", func() {
 			framework.ExpectNoError(err, "failed to check status")
 		})
 
+		ginkgo.It("Deploy and delete cluster across kubernetes", func() {
+			ns1 := namespaces[0]
+			ns2 := namespaces[1]
+
+			cluster1Domain := defaultClusterDomain
+			cluster2Domain := defaultClusterDomain
+
+			tc1 := GetTCForAcrossKubernetes(ns1, "basic-1", version, cluster1Domain, nil)
+			tc2 := GetTCForAcrossKubernetes(ns2, "basic-2", version, cluster2Domain, tc1)
+
+			tc1.Spec.PD.Replicas = 3
+			tc1.Spec.TiKV.Replicas = 3
+
+			ginkgo.By("Deploy the basic cluster-1")
+			utiltc.MustCreateTCWithComponentsReady(genericCli, oa, tc1, 10*time.Minute, 10*time.Second)
+
+			ginkgo.By("Deploy the basic cluster-2")
+			utiltc.MustCreateTCWithComponentsReady(genericCli, oa, tc2, 10*time.Minute, 10*time.Second)
+
+			// connectable test
+			_, err := utiltidb.TiDBIsConnectable(fw, tc1.Namespace, tc1.Name, "root", "")()
+			framework.ExpectNoError(err, "tc1 is not connectable")
+			_, err = utiltidb.TiDBIsConnectable(fw, tc2.Namespace, tc2.Name, "root", "")()
+			framework.ExpectNoError(err, "tc2 is not connectable")
+
+			ginkgo.By("Scale in cluster-2, and delete the cluster-2")
+
+			framework.ExpectNoError(controller.GuaranteedUpdate(genericCli, tc2, func() error {
+				tc2.Spec.PD.Replicas = 0
+				tc2.Spec.TiDB.Replicas = 0
+				tc2.Spec.TiKV.Replicas = 0
+				tc2.Spec.TiFlash.Replicas = 0
+				tc2.Spec.TiCDC.Replicas = 0
+				tc2.Spec.Pump.Replicas = 0
+				return nil
+			}), "failed to scale in cluster 2")
+			err = wait.PollImmediate(10*time.Second, 6*time.Minute, func() (bool, error) {
+				if err := CheckPeerMembersAndClusterStatus(genericCli, tc1, tc2); err != nil {
+					log.Logf("wait for tc2 member deleted, status: %s", err)
+					return false, nil
+				}
+				return true, nil
+			})
+			framework.ExpectNoError(err, "tc1 is not all healthy")
+			framework.ExpectNoError(genericCli.Delete(context.TODO(), tc2), "failed to delete cluster 2")
```

Comment:
how about checking `tc1` healthy again after tc2 is deleted?

Reply:
fixed, added a connection test and tc status check.